pnda_env_example.yaml

```diff
@@ -233,3 +233,11 @@ datanode:
   DATA_VOLUME_COUNT: 1
   # DEVICE_ROOT sets the disk device root name
   DEVICE_ROOT: xvdb
+
+kafka:
```

Comment:
rename DATA_DIRS and DEVICE_ROOT to KAFKA_DATA_DIRS and KAFKA_DEVICE_ROOT, as it stands DEVICE_ROOT will clash the one used under the datanode section. I will also create a JIRA to rename the datanode settings to be more specific too.